### PR TITLE
Update version to v1.4.1

### PR DESCRIPTION
**Pull request by yymao**
Update version for a new release.

### CODE REVIEW

**Comment by yymao** [comment]
@JoanneBogart I'd like to merge this after #603 is merged, unless there's any objection.

**Review by JoanneBogart — APPROVED** [review]
It's fine.   Sorry I didn't see this sooner.

**Pull request merged**
